modules/ting_search_carousel/ting_search_carousel.module

```diff
@@ -288,8 +288,10 @@ function _ting_search_carousel_find_entities_with_covers($query, $start, $size,
     'enrich' => TRUE,
     'allObjects' => FALSE,
   );
+
+  // TODO: use the Search Abstraction Layer (ting_start_query()).
   // Ting pages are 1, not 0, based.
-  $result = ting_do_search($query, ($start++) + 1, $size, $options);
+  $result = opensearch_do_search($query, ($start++) + 1, $size, $options);
```

Comment:
This seems odd:

- Are we not going to port `ting_search_carousel()`?
- If not then we should load the `opensearch` client

Reply:
I can't find an issue for it, but we should have one as it required - maybe we've missed it?

Reply:
I've created BBS-178